sbom-utility-scripts/scripts/sbom-for-modelcar-task/test_data/cyclonedx.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "http://cyclonedx.org/schema/bom-1.5.schema.json",
+  "bomFormat": "CycloneDX",
+  "specVersion": "1.5",
+  "version": 1,
+  "metadata": {
+    "component": {
+      "type": "container",
+      "name": "aipcc/modelcar-image",
+      "purl": "pkg:oci/aipcc/modelcar-image@8ef392004884ee00ccaabd279ea6859fc021c7b613f3103f568277f078764998?repository_url=repository.example.com/aipcc/modelcar-image",
+      "version": "v1.0",
+      "hashes": [
+        {
+          "alg": "SHA-256",
+          "content": "8ef392004884ee00ccaabd279ea6859fc021c7b613f3103f568277f078764998"
+        }
+      ]
+    }
+  },
+  "components": [
+    {
+      "type": "container",
+      "name": "aipcc/modelcar-image",
+      "purl": "pkg:oci/aipcc/modelcar-image@8ef392004884ee00ccaabd279ea6859fc021c7b613f3103f568277f078764998?repository_url=repository.example.com/aipcc/modelcar-image",
+      "version": "v1.0",
+      "hashes": [
+        {
+          "alg": "SHA-256",
+          "content": "8ef392004884ee00ccaabd279ea6859fc021c7b613f3103f568277f078764998"
+        }
+      ]
+    },
+    {
```

Comment:
To represent that "aipcc/modelcar-image" descends from both "aipcc/base-image" and "aipcc/model-image", I wonder if the structure ought to be like:
```
"components": {
  {
    "type": "container",
    "name": "aipcc-modelcar-image",
    ...
    "components": [
        {
            "type": "container",
            "name": "aipcc/base-image",
            ...
        },
        {
            "type": "container",
            "name": "aipcc/model-image",
            ...
        }
    ]
}
```

Reply:
I've updated the structure to accommodate the suggestion, please could you review it again?